editor/src/clj/editor/graphics.clj

```diff
@@ -314,7 +314,12 @@
           :semantic-type :semantic-type-normal
           :coordinate-space :coordinate-space-world
           :data-type :type-float
-          :element-count 3}]))
+          :element-count 3}
+         {:name "tangent"
+          :semantic-type :semantic-type-tangent
+          :coordinate-space :coordinate-space-world
+          :data-type :type-float
+          :element-count 4}]))
```

Comment:
When using world-space meshes, the tangents in `:tangent-data` will be transformed by the `renderable-data->world-direction-v4` function. Currently, this writes a zero for the W component when it calls the `geom/transf-n4` function. I think we need to update `geom/transf-n4` to pass along the unaltered W component.

It seems to me, we would need to do something similar on the engine side since we presumably transform the tangents using the normal matrix somewhere there as well, but I'm not seeing it in this PR? 🤔

Reply:
The engine preserves the handedness in Rig.cpp in the "GenerateNormalData" function.